interfaces/cython/cantera/test/test_convert.py

```diff
@@ -232,14 +232,22 @@ def test_explicit_reverse_rate(self):
         self.checkKinetics(ref, gas, [300, 800, 1450, 2800], [5e3, 1e5, 2e6])
 
         # Reactions with explicit reverse rate constants are transformed into
-        # two irreversible reactions with reactants and products swapped.
+        # two irreversible reactions with reactants and products swapped, unless
+        # the explicit reverse rate is zero so only the forward reaction is used.
         Rr = gas.reverse_rate_constants
         self.assertEqual(Rr[0], 0.0)
         self.assertEqual(Rr[1], 0.0)
+        self.assertEqual(Rr[2], 0.0)
+        self.assertEqual(Rr[3], 0.0)
+        self.assertEqual(Rr[4], 0.0)
         Rstoich = gas.reactant_stoich_coeffs()
         Pstoich = gas.product_stoich_coeffs()
-        self.assertEqual(list(Rstoich[:,0]), list(Pstoich[:,1]))
-        self.assertEqual(list(Rstoich[:,1]), list(Pstoich[:,0]))
+        self.assertEqual(list(Rstoich[:, 0]), list(Pstoich[:, 1]))
+        self.assertEqual(list(Rstoich[:, 1]), list(Pstoich[:, 0]))
+        self.assertEqual(list(Rstoich[:, 2]), list(Pstoich[:, 3]))
+        self.assertEqual(list(Rstoich[:, 3]), list(Pstoich[:, 2]))
+        self.assertEqual(list(Rstoich[:, 4]), [1, 0, 0, 0, 1, 0, 0])
```

Comment:
Isn't this testing that the reactants of reaction 1 are products of reaction 2 and vice versa?  It seems like the fact that its using stoichiometric coefficients is incidental and quite convenient. Do you mean the reactant/product symbols should be compared?